core/baseline-core-v1.0-psd01.md

```diff
@@ -3001,60 +3003,227 @@ This section of the document focuses on the requirements and considerations rela
 
 ## 8.1 Internal Authoritative Data for BPIs
 
+###### intauthbpidata
+
 Internal authoritative data means that there exists only one authoritative version of the input data in some system of record of a BPI participant, making it the only choice for input into a BPI workstep. Authoritative in this context means that BPI participants involved in a BPI workstep have agreed that the authoritative source is the accurate and reliable truth for that particular type of data.
 
 #### **[O10]** 
 Internal authoritative input data to a BPI workstep MAY come from a single source.
 
-[[D10]](#d10) Testability: Since the [Baseline Protocol API specification is testable](https://github.com/eea-oasis/baseline-standard/blob/main/api/baseline-api-v1.0-psd01.md), and since internal authoritative input data with source identification will be used in the Baseline Protocol APIs based on agreed upon data schames between counterparties, the requirement is testable. 
+[[D10]](#d10) Testability: 
+
+Preconditions:
+
+* A PBI test system is up and running.
+* There is a given set of test BPI Subjects that are part of a test workgroup.
+* The test BPI participants in a workgroup have agreed to allow internal authoritative input data to a BPI workstep only from a single, specific data source.
+* The test workgroup is associated with only one test workstep.
+* The test workstep accepts only input data from the agreed upon data source for a given data schema, and the input data is transformed in an agreed upon manner to an expected output.
```

Comment:
This is two preconditions in one. The first:

> The test workstep accepts only input data from the agreed upon data source for a given data schema

seems to set as a precondition the third passing criterion for this test process. In any event:

```suggestion
* The test workstep accepts only input data from the agreed upon data source for a given data schema
* The test workstep transforms the input data in an agreed upon manner to an expected output.
```

(But I don't think the second requirement is necessary here. The correctness of the transformation according to the agreement between parties is already covered elsewhere in the spec)

Reply:
Removed 2nd precondition and removed 1st passing criteria. That should do it.